tests/paz/pipelines/pix2pose_metric_test.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import cv2
+import os
+from tensorflow.keras.utils import get_file
+from paz.backend.image import load_image
+from paz.backend.camera import Camera
+from paz.pipelines import PIX2POSEPowerDrill
+from paz.backend.groups.quaternion import quaternion_to_rotation_matrix, \
+    rotation_matrix_to_quaternion
+import numpy as np
+from paz.backend.groups import build_rotation_matrix_x
+from paz.abstract.messages import Pose6D
+from examples.pix2pose.eval import compute_ADI, compute_ADD, \
+    transform_points3D, get_translation, get_quaternion
```

Comment:
Please move this test inside examples/pix2pose/ directory and change the name to  pose_metrics_test.py. Change the import accordingly such that you can call pytest paz/examples/pix2pose/pose_metrics_test.py

Reply:
resolved